components/html-block/html-block.js

```diff
@@ -242,6 +245,10 @@ class HtmlBlock extends LitElement {
 		}
 	}
 
+	async getLoadingComplete() {
```

Comment:
Oh I like this! There are use cases in the LMS where it would be useful to know when HTML has finished rendering.